src/google/protobuf/compiler/command_line_interface.cc

```diff
@@ -118,6 +100,14 @@ namespace compiler {
 namespace {
 #if defined(_WIN32) && !defined(__CYGWIN__)
 static const char* kPathSeparator = ";";
+// DO NOT include <io.h>, instead create functions in io_win32.{h,cc} and import
+// them like we do below.
+using google::protobuf::stubs::access;
```

Comment:
Can you use the google::protobuf::internal namespace? That's what we currently use for most other stubs headers.